models/index.js

```diff
@@ -0,0 +1,13 @@
+const Sequelize = require('sequelize')
+const FootballModel = require('./football')
+
+const connection = new Sequelize('football', 'football', 'F00tB4LL!', {
+  host: 'localhost',
+  dialect: 'mysql'
+})
+const Teams = FootballModel(connection, Sequelize)
+const Op = Sequelize.Op
+module.exports = {
+  Teams,
+  Op
```

Comment:
Oh I kind of like this. Fewer places to import the library. I might steal this 😄